FundParser/FundParser.BL.Tests/Services/DownloaderServiceTests.cs

```diff
@@ -36,18 +36,18 @@ public async Task DownloadTextFileAsString_InvalidLink_ReturnsNull()
                     Content = null
                 });
             var httpClient = new HttpClient(new MockHttpMessageHandler(_messageHandlerMock.Object));
-
-            // Act
             var downloaderService = new DownloaderService(_loggerMock.Object, httpClient);
-            var result = await downloaderService.DownloadTextFileAsStringAsync(urlInvalid, _validHeaders, CancellationToken.None);
+            // Act
```

Comment:
Since we are using constraint asserts everywhere, this would be better as:
```
            // Act, Assert
            Assert.That(async () => await downloaderService.DownloadTextFileAsStringAsync(urlInvalid, _validHeaders, CancellationToken.None), Throws
                .TypeOf<UriFormatException>()
                .With
                .Message.EqualTo("Invalid URI: The format of the URI could not be determined."));
```
But it would have to be changed in all these exception throwing tests.